### ISSUE TITLE
Jupyter lab: "Error displaying widget: model not found"

### ISSUE THREAD

**Issue by jlaw9** [fekad/jupyter-jsmol#37]
I followed the installation instructions for jupyter lab, but I'm unable to get jupyter-jsmol to show any structures. I only get the error `Error displaying widget: model not found`.

Any suggestions for how to get it working?

Here's some sample code: 
```
from jupyter_jsmol import JsmolView
from ipywidgets import Layout, widgets, interact
!curl -OL https://raw.githubusercontent.com/fekad/jupyter-jsmol/master/examples/data/c2h410.xyz
view1 = JsmolView.from_file("c2h410.xyz") 
view1
```

Here's the output of `conda list` for my jupyter packages:

```
jupyter                   1.0.0                    py38_7
jupyter-bokeh             3.0.4                    pypi_0    pypi
jupyter-jsmol             2021.9.0                 pypi_0    pypi
jupyter-server            1.8.0                    pypi_0    pypi
jupyter-server-proxy      3.1.0                    pypi_0    pypi
jupyter_client            7.0.1              pyhd3eb1b0_0
jupyter_console           6.4.0              pyhd3eb1b0_0
jupyter_core              4.8.1            py38h06a4308_0
jupyterlab                3.0.16                   pypi_0    pypi
jupyterlab-server         2.5.2                    pypi_0    pypi
jupyterlab_pygments       0.1.2                      py_0
jupyterlab_widgets        1.0.0              pyhd3eb1b0_1
```

**Comment by jlaw9** [comment]
I also tried following the steps for a fresh development install in the README and am getting the similar message: "Error displaying widget"

**Comment by ltalirz** [comment]
Hey @fekad, long time no see ;-) Happy new year!

I'm getting the same error on your mybinder repo when launched with [this link](https://gesis.mybinder.org/binder/v2/gh/fekad/jupyter-jsmol/master)

![image](https://user-images.githubusercontent.com/1053245/148090433-ba9a3ccc-1670-4a75-9252-2299ba9e8c16.png)

The default mybinder launches a standard jupyter notebook server, where the widget also does not display (but shows no error message)

![image](https://user-images.githubusercontent.com/1053245/148090754-00151ee9-bc88-4711-816b-bb11f655bab0.png)


Messages in the javascript console
![image](https://user-images.githubusercontent.com/1053245/148090938-d0bbfc00-2a99-4649-b1e9-8c3b81f86ee1.png)

**Comment by fekad** [comment]
@jlaw9 @ltalirz I started to fix the issue in the master branch. 
- I rolled back the installation process (using JS library and older version of [jupyter-packing](https://github.com/jupyter/jupyter-packaging))
- The other major issue was caused by a variable name conflict between the "new" version of the notebook and the JSmol code. (https://sourceforge.net/p/jmol/patches/38/)
- the proper licences have to be clarified (#36) before I can do a new release.
